Userland/Libraries/LibGfx/CMakeLists.txt

```diff
@@ -41,4 +43,4 @@ set(SOURCES
 )
 
 serenity_lib(LibGfx gfx)
-target_link_libraries(LibGfx LibM LibCompress LibCore LibTextCodec)
+target_link_libraries(LibGfx LibM LibCompress LibCore LibTextCodec LibRemoteGfx)
```

Comment:
LibGfx and LibRemoteGfx have a circular dependency the way you've set this up :/

Reply:
And this is bad because....?

Reply:
Won't it be impossible to build this from scratch, then? Try a clean build.

Reply:
I believe that ninja can handle it for linux and Serenity builds. For Lagom, circular library dependencies are forbidden on macOS, enforced by the linker. It also makes a makefile generator build not happy. Luke ran into some issues with a circular dependency b/w LibCore, LibCrypto and LibCrypt trying to build Serenity on Serenity yesterday (ninja Port is borkd)

Reply:
Why would it? They're not statically linked against each other where there would indeed be a chicken-egg problem. I've done countless clean builds from scratch with this and haven't had any issues whatsoever.

Reply:
Hm so then what should we do? Merge it into LibGfx like with TTF?

Reply:
I think merging it would be reasonable. Seems like a questionable architecture decision to me to introduce circular library dependencies, even if it happens to works at runtime.

Reply:
I moved it into LibGfx